pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -1693,12 +1699,10 @@ func Convert_v1_VirtualMachineInstance_To_api_Domain(vmi *v1.VirtualMachineInsta
 		domain.Spec.Clock = newClock
 	}
 
-	if vmi.Spec.Domain.Features != nil {
-		domain.Spec.Features = &api.Features{}
-		err := Convert_v1_Features_To_api_Features(vmi.Spec.Domain.Features, domain.Spec.Features, c)
-		if err != nil {
-			return err
-		}
+	domain.Spec.Features = &api.Features{}
```

Comment:
Just out of curiosity... Why did you decide to initiate the features struct here and not in the `SetObjectDefaults_Domain` - in pkg/virt-launcher/virtwrap/api/defaults.go ?

Reply:
Yeah, I think your suggestion is better, I moved it to over there :+1: 
Thanks!